src/dialogs/config-flow/show-dialog-options-flow.ts

```diff
@@ -110,9 +110,11 @@ export const showOptionsFlowDialog = (
       },
 
       renderShowFormStepFieldError(hass, step, error) {
-        return hass.localize(
-          `component.${configEntry.domain}.options.error.${error}`,
-          step.description_placeholders
+        return (
+            hass.localize(
+                `component.${configEntry.domain}.options.error.${error}`,
+                step.description_placeholders
+            ) || error
```

Comment:
```suggestion
        return (
          hass.localize(
            `component.${configEntry.domain}.options.error.${error}`,
            step.description_placeholders
          ) || error
```

Whitespace needs to be exact to satisfy the lint check, it's somewhat tricky if you don't have a development environment setup.